web/app/js/components/ConfigureProfilesMsg.jsx

```diff
@@ -89,46 +117,75 @@ class ConfigureProfilesMsg extends React.Component {
           variant="outlined"
           color="primary"
           size="small"
-          onClick={this.handleClickOpen}>Create Service Profile
+          onClick={this.handleClickOpen}>
+          Create Service Profile
         </Button>
       );
     }
+
     return (
       <React.Fragment>
         {button}
-
         <Dialog
           open={this.state.open}
           onClose={this.handleClose}
           aria-labelledby="form-dialog-title">
           <DialogTitle id="form-dialog-title">New service profile</DialogTitle>
           <DialogContent>
             <DialogContentText>
-              To create a service profile, download a profile and then apply it with `kubectl apply`.
+              To create a service profile, download a profile and then apply it
+              with `kubectl apply`.
             </DialogContentText>
-            <TextField
-              id="service"
-              label="Service"
+            <FormControl
               className={classes.textField}
-              value={this.state.service}
-              onChange={this.handleChange('service')}
-              margin="normal" />
-            <TextField
-              id="namespace"
-              label="Namespace"
+              onBlur={() => this.validateFields('service', query.service)}
+              error={error.service}>
+              <InputLabel htmlFor="component-error">Service</InputLabel>
+              <Input
+                id="component-error"
+                value={this.state.service}
+                onChange={this.handleChange('service')}
+                aria-describedby="component-error-text" />
+              {error.service && (
+                <FormHelperText id="component-error-text">
+                  The Service name must consist of lower case alphanumeric
+                  characters or &#45; and must start and end with an
+                  alphanumeric character
+                </FormHelperText>
+              )}
+            </FormControl>
+            <FormControl
               className={classes.textField}
-              value={this.state.name}
-              onChange={this.handleChange('namespace')}
-              margin="normal" />
+              onBlur={() => this.validateFields('namespace', query.namespace)}
+              error={error.namespace}>
+              <InputLabel htmlFor="component-error">Namespace</InputLabel>
+              <Input
+                id="component-error"
+                value={this.state.name}
+                onChange={this.handleChange('namespace')}
+                aria-describedby="component-error-text" />
+              {error.namespace && (
+                <FormHelperText id="component-error-text">
+                  The Namespace name must consist of lower case alphanumeric
```

Comment:
the service and namespace validation errors are slightly different.

```
	// a DNS-1035 label must consist of lower case alphanumeric characters or '-',
	// start with an alphabetic character, and end with an alphanumeric character
	if errs := validation.IsDNS1035Label(options.name); len(errs) != 0 {
		return fmt.Errorf("invalid service %q: %v", options.name, errs)
	}

	// a DNS-1123 label must consist of lower case alphanumeric characters or '-',
	// and must start and end with an alphanumeric character
	if errs := validation.IsDNS1123Label(options.namespace); len(errs) != 0 {
		return fmt.Errorf("invalid namespace %q: %v", options.namespace, errs)
	}
```

I'd recommend the following error text:
Service:
`Service name must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character`
Namespace:
`Namespace must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character`

Of course, we could also go with "Must be a valid Kubernetes service name" but that's less informative to the user.